src/schema/v2/homeView/__tests__/HomeViewSection.test.ts

```diff
@@ -478,4 +478,44 @@ describe("HomeViewSection", () => {
       `)
     })
   })
+
+  describe("ActivityRailHomeViewSection", () => {
+    it("returns the latest activity", async () => {
+      const query = `
+        {
+          homeView {
+            section(id: "home-view-section-latest-activity") {
+              __typename
+              ... on ActivityRailHomeViewSection {
+                component {
+                  title
+                }
+              }
+            }
+          }
+        }
+      `
+
+      const context = {
+        authenticatedLoaders: {
+          meLoader: jest.fn().mockReturnValue({ type: "User" }),
+        },
+      }
+
+      const data = await runQuery(query, context)
+
+      expect(data).toMatchInlineSnapshot(`
+        Object {
+          "homeView": Object {
+            "section": Object {
+              "__typename": "ActivityRailHomeViewSection",
+              "component": Object {
+                "title": "Latest Activity",
+              },
+            },
```

Comment:
**question (non-blocking):** Since you eventually updated the data resolution (👏🏽) do want to reflect that in the spec as well?

Reply:
makes sense - I will update the specs